src/Cli/dotnet/Telemetry/TopLevelCommandNameAndOptionToLog.cs

```diff
@@ -35,12 +35,34 @@ public List<ApplicationInsightsEntryFormat> AllowList(ParseResult parseResult, D
                         new Dictionary<string, string>
                         {
                             { "verb", topLevelCommandName},
-                            { option.Name, parseResult.GetValueForOption(option)?.ToString() }
+                            { option.Name, Stringify(parseResult.GetValueForOption(option)) }
```

Comment:
this looks fine to me, @nohwnd are you aware to some dashboard using this information on our side?

Reply:
Ok so this update looks good.